util/json_patch.py

```diff
@@ -20,16 +20,33 @@ def _assert(condition, msg):
         raise PatchException(msg)
 
 
-def create_entity(properties_values, entity_class=None):
+def create_entity(properties_values, method_define_entity=None):
     """Create new entity of class specified."""
-    if entity_class:
-        entity = entity_class()
+    if method_define_entity:
```

Comment:
se for None, não faz nada?

Reply:
Ele simplesmente retorna o proprio dicionário de valores (properties_values), pois se ele não passar o método não há como saber que tipo de entidade será, então retorna o próprio dicionário recebido.